silx/gui/plot/items/roi.py

```diff
@@ -392,8 +399,14 @@ def _createPlotItems(self):
                 item.setColor(color)
                 item.setVisible(self.isVisible())
                 plot._add(item)
+                # connect item changed
                 item.sigItemChanged.connect(functools.partial(
                     self._controlPointAnchorChanged, index))
+                # connect items pressed and released signals
+                item.sigDragStarted.connect(functools.partial(
```

Comment:
One final comment, the `functools.partial` is actually not used as the parameters of `_editingStarted|Finished` are not used, probably best to remove unless you want to give the index in `sigEditingStarted|Finished` which can be a good idea if it is meaningful